frontend/src/app/app.tsx

```diff
@@ -26,7 +28,7 @@ const App: React.FC = () => {
     }, [isRoot, dispatch]);
 
     return (
-        <>
+        <ErrorBoundary fallback={<div>Something went wrong</div>}>
```

Comment:
It would be nice to have some styles so that the fallback text is centered on the screen

Reply:
done